reconcile/gitlab_housekeeping.py

```diff
@@ -196,17 +198,34 @@ def get_merge_requests(dry_run: bool, gl: GitLabApi) -> list:
                 gl.remove_label_from_merge_request(mr.iid, LGTM)
             continue
 
+        label_events = mr.resourcelabelevents.list()
+        approval_found = False
+        for label in reversed(label_events):
+            if label.action == "add":
+                label_name = label.label["name"]
+                added_by = label.user["username"]
+                if users_allowed_to_label:
+                    if added_by not in users_allowed_to_label:
```

Comment:
Can be in one line and would be easier to read :)

Reply:
thanks! https://github.com/app-sre/qontract-reconcile/pull/2653/commits/4a06dcc73f37c9d2fa4c0b28faea43ffa96d15c5